src/gsi/setupq.f90

```diff
@@ -317,6 +318,34 @@ subroutine setupq(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_diagsav
   do i=1,nobs
      muse(i)=nint(data(iuse,i)) <= jiter
   end do
+!  If HD raobs available move prepbufr version to monitor
+  nhdq=0
```

Comment:
@jderber-NOAA Last question: why does nhdq get initialized to zero while nhdt and the others do not?

Reply:
You are correct.  I had found this error a few days ago and am awaiting instructions from Mike how to change it so I do not mess things up.

Reply:
Changes have been made to the code to address this  issue and the one before it.